cmd/builder/.goreleaser.yml

```diff
@@ -8,7 +8,7 @@ builds:
   - flags:
       - -trimpath
     ldflags:
-      - -s -w
+      - -s -w -X go.opentelemetry.io/collector/cmd/builder/internal.version={{ .Version }}
```

Comment:
@jackgopack4 have you been able to test this change?

Reply:
nvm, I forgot this is the same string it used to be

Reply:
yes, though, I did a few releases in github.com/jackgopack4/opentelemetry-collector-releases
https://github.com/jackgopack4/opentelemetry-collector-releases/releases/tag/cmd%2Fbuilder%2Fv0.107.1
for example